### ISSUE TITLE
core-decorators.js is in .gitignore but is tracked

### ISSUE THREAD

**Issue by Ephys** [jayphelps/core-decorators.js#105]
.gitignore contains a ligne that ignores any file called `core-decorators.js`
https://github.com/jayphelps/core-decorators.js/blob/master/.gitignore#L3

But there is one, inside `src`, which is currently tracked
https://github.com/jayphelps/core-decorators.js/blob/master/src/core-decorators.js

Any guess on which one is right?

**Comment by jayphelps** [comment]
Good catch! Thanks https://github.com/jayphelps/core-decorators.js/commit/b69df7b09a8784e69b8050feaad689ddc95b6461
